### PR TITLE
Add Travis CI logo

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**

### CODE REVIEW

**Comment by paulo-ferraz-oliveira** [comment]
Let's wait until Travis CI integration is set up for this PR to make sense.

**Comment by paulo-ferraz-oliveira** [comment]
> I'd propose we also adapt the minimum supported version to some reasonable value (it's not set, but Travis CI is set to run from R16B03-1 onward), in `rebar.config` and adapt `.travis.yml` to reflect that (up until 23.0).

I did this in another PR.

File: README.md
Comment (g-andrade) [inline README.md:6]: ^ Very cool, didn't know about these.

Comment (paulo-ferraz-oliveira) [reply README.md:6]: I think I found one of these in `elli-lib`.

**Review by lrascao — APPROVED** [review]
good to go after rebase

**Comment by paulo-ferraz-oliveira** [comment]
On it.

**Review by g-andrade — APPROVED** [review]

**Pull request merged**
